doc/faq.md

```diff
@@ -1,6 +1,209 @@
+
+<a id="monitor" />
+
+### How to monitor or get statistics of DPVS ?
+
+You can use `ipvsadm` and `dpip` tools. For example,
+
+```
+$ ipvsadm -ln --stats
+$ dpip -s link show
+$ dpip -s link show cpu
+```
+
+For example, to get the throughput for each VIP/RS, you can use `ipvsadm -ln --stats`, and `ipvsadm -Z` to clear the statistics.
+
+Note `--rate` option of `ipvsadm` is not supported.
+
+It may need to write scripts to parse the outputs or integrated with your local admin-system.
+
+<a id="support" />
+
+### Where can I found the support ? if doc not helps.
+
+If any question, pls make sure you have read the `doc/` first, and `LVS`'s [documents](https://github.com/alibaba/LVS/tree/master/docs) are also helpful. It's the better to have some experience about networking configuration (e.g., routing, neigbour, ...) since DPVS is kernel-bypass, basic routing, IP address configurations are needed to be setup from scratch.
+
+We have Chinese `QQ-Group` or `WeChat-Group`, you can ask questions, raise issues, talk about design, help others or discuss everything else about DPVS. Here's the [entry](https://github.com/iqiyi/dpvs/issues/147) for WeChar-Group (微信群) and [entry](https://github.com/iqiyi/dpvs/issues/83) for QQ-Group (QQ群). Email to `qlb-devel #at# qiyi #dot# com` is another way.
+
+At last, remember, you can find answer of all kind of questions from the codes, DPVS is open-sourced :).
+
+<a id="test" />
+
+### How to test the performance of DPVS ?
+
+We use `wrk` as HTTP client and `f-stack/nginx` as Real Server. When we testing the performance of DPVS, at least 6 physical machines are used as Client (`wrk`), and 4 physical machines for `f-stack/nginx`.
+
+For the machine running `wrk`, IRQ affinity should be set to make sure all CPUs are used. You can find the scripts like `set_irq_affinity_ixgbe` from Internet.
+
+To get the test result for QPS, pls check the output of `wrk`. To calculate *Packet Per-Second* (`pps`), *Bytes Per-Second* (`bps`), you can use `ipvsadm -ln --stats` or `dpip -s link show`.
+
+We have tested 10G NIC only, and the result shows DPVS can reach the line-speed of 10G NIC with small packets. We have not test 25G/40G/100G NIC yet. Although it's in plan.
+
+The test result can be found in `README.md`. We have no chance to use professional instruments for test.
+
+<a id="vir-dev" />
+
+### Is DPVS support Bonding/VLAN/Tunnel ?
+
+Yes. To use bonding device, pls check `conf/dpvs.bond.conf.sample`. To set up VLAN/Tunnel device, you can refer `tutorial.md` or `dpip vlan help`.
+
+<a id="config-dpvs" />
+
+### How to configure DPVS ?
+
+There're several ways:
+
+* `/etc/dpvs.conf`
+
+You can modify `dpvs.conf`, and refer `conf/xxx.sample`. Some parameters are configurable during
+ run-time (on-fly), while others are configurable in initialization stage only. Pls use `kill -SIGHUP <DPVS-PID>` to reload the former on-fly kind of parameters in `dpvs.conf`.
```

Comment:
....are configurable in initialization stage only. Refer to `conf/dpvs.conf.items` for all available parameters  and corresponding type, default value and supported value range. ....